package.json

```diff
@@ -9,7 +9,9 @@
   "repository": "https://github.com/profusion/validated-env-schema",
   "lint-staged": {
     "*.{ts, js}": [
-      "eslint",
+      "prettier --write",
+      "eslint --fix",
+      "git add",
```

Comment:
this `git add` step is no longer needed, just the `prettier` and `eslint`

Reply:
Done.